rebalance/rebalance.py

```diff
@@ -398,9 +403,9 @@ def check_liquidity_threshold(channels: list, threshold: Millisatoshi):
 
 
 def binary_search(channels: list, low: Millisatoshi, high: Millisatoshi):
-    if high - low < Millisatoshi("1sat"):
-        return low
     next_step = (low + high) / 2
+    if high - low < Millisatoshi("1sat"):
+        return next_step
     if check_liquidity_threshold(channels, next_step):
         return binary_search(channels, next_step, high)
```

Comment:
Unrelated to this PR:
Python does not support proper tail recursions. Maybe we should rewrite this to a loop.

Update:
If I see this correct, for  the biggest channel being i.e. `0.05btc` this adds up to 23 recursions until the value goes below 1sat. Not sure if thats a big deal or not, likely not a big deal :D